dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/servlet/DhisWebApiWebAppInitializer.java

```diff
@@ -95,6 +95,12 @@ private DhisConfigurationProvider getConfig() {
   public static void setupServlets(
       ServletContext context, AnnotationConfigWebApplicationContext webApplicationContext) {
 
+    context
+        .addFilter(
+            "SpringSessionRepositoryFilter",
+            new DelegatingFilterProxy("springSessionRepositoryFilter"))
```

Comment:
Does it matter that the strings here are not the same? One is lower case S and one upper.

Reply:
This is just naming convention for filters.